tensorflow-ops/src/TensorFlow/Ops.hs

```diff
@@ -256,6 +257,15 @@ constant (Shape shape') values
                       [def & TensorShape.size .~ x | x <- shape']
                 & tensorVal .~ values
 
+-- | Reshape a N-D tensor down to a scalar.
+toScalar :: (TensorType a) 
```

Comment:
This should probably be named scalarize or something similar (extractScalar?). We already have "unScalar".